hugolib/author.go

```diff
@@ -43,3 +50,84 @@ type Author struct {
 // - linkedin
 // - skype
 type AuthorSocial map[string]string
+
+// URL is a convenience function that provides the correct canonical URL
+// for a specific social network given a username. If an unsupported network
+// is requested, only the username is returned
+func (as AuthorSocial) URL(key string) string {
+	switch key {
+	case "github":
+		return "https://github.com/" + as[key]
```

Comment:
Never mind -- that can be handled in a separate issue. But we should provide that.